File: FLUTTER/igest_app/lib/models/principal/pais.dart
class Pais {
  int idPais;
  String nomPais;
  bool activo;
  int orden;

  Pais({
    required this.idPais,
    required this.nomPais,
    required this.activo,
    required this.orden,
  });

  factory Pais.fromJson(Map<String, dynamic> json) {
    return Pais(
      idPais: json['id_pais'],
      nomPais: json['nom_pais'],
      activo: json['activo'],
      orden: json['orden'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'id_pais': idPais,
      'nom_pais': nomPais,
      'activo': activo,
      'orden': orden,
    };
  }
}

File: SPRING/igest.server/src/main/java/com/yupanastudio/igest/server/principal/model/Pais.java
/*
 * Document   : Pais
 * Author     : Franklin Barzola Olivares <frankoantonio@gmail.com>
 */
package com.yupanastudio.igest.server.principal.model;

import org.springframework.data.annotation.Id;

public class Pais {

    @Id
    private Integer idPais;
    private String nomPais;
    private Boolean activo;
    private int orden;

    public Integer getIdPais() {
        return idPais;
    }

    public void setIdPais(Integer idPais) {
        this.idPais = idPais;
    }

    public String getNomPais() {
        return nomPais;
    }

    public void setNomPais(String nomPais) {
        this.nomPais = nomPais;
    }

    public Boolean getActivo() {
        return activo;
    }

    public void setActivo(Boolean activo) {
        this.activo = activo;
    }

    public int getOrden() {
        return orden;
    }

    public void setOrden(int orden) {
        this.orden = orden;
    }

}
